src/particle.cpp

```diff
@@ -83,6 +83,8 @@ Particle::create_secondary(Direction u, double E, Type type) const
   bank.r = this->r();
   bank.u = u;
   bank.E = settings::run_CE ? E : g_;
+
+  n_bank_second_ += 1;
```

Comment:
Rather than incrementing the number of particles added to the secondary bank, why don't we just increment the energy produced (thereby avoiding the loop in tally_scoring.cpp over secondary bank particles)? I suppose we'd still need the special case for positrons here, but it simplifies the logic for tallying. What do you think?

Reply:
@paulromano I thought the variable `n_bank_second_` (1) has a minimal influence on the transport process, (2) is more general and may be used for tallies other than getting the energy of the secondary particles. Anyway, I can replace it with energy if you think it is better.